scripts/linters/general_purpose_linter.py

```diff
@@ -409,27 +402,12 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
-    {
-        'regexp': re.compile(r'self.assertEquals\('),
-        'message': 'Please do not use self.assertEquals method. ' +
-                   'This method has been deprecated. Instead use ' +
-                   'self.assertEqual method.',
-        'excluded_files': (),
-        'excluded_dirs': ()
-    },
     {
         'regexp': re.compile(r'with open\(|= open\('),
```

Comment:
Can we move this to the .pylintrc `open=>python_utils.open_file`?

Reply:
Done!